# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../clip/hotfix_clip_file.c.patch
# Copyright (C) 2005 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- clip-prg-1.1.15-1.orig/clip/file.c	2005-08-25 04:36:33.000000000 -0400
+++ clip-prg-1.1.15-1/clip/file.c	2005-12-30 09:35:11.000000000 -0300
@@ -1965,7 +1965,7 @@
 	{
 		char *mp;
 		long j, modlen, size;
-		char *name = strdup(file->name), *upname;
+		char *name = strdup(file->mname), *upname;
 		char *s;
 
 		s = strchr(name, '.');
@@ -2061,7 +2061,7 @@
 	{
 		char *mp;
 		long j, k, modlen, size;
-		char *name = strdup(file->name);
+		char *name = strdup(file->mname);
 		char *s;
 
 		s = strchr(name, '.');
